### DISCUSSION TITLE
One click action button, batch action

### DISCUSSION THREAD

**Discussion by joeblowout** [NginxProxyManager/nginx-proxy-manager#2193] [Ideas]
I really like npm, been using it since last June 2021, it is really convenient. When proxy hosts get too many however, it starts to become a hassle to manage.

Would it be possible to add action button to perform batch action? For example, update all certificates at once, check if domain names are reachable, remove all unreachable proxy hosts etc. Adding checkboxes for each proxy box to decide whether to include or exclude for batch action would be great too!

Thank you for such wonderful tool!

**Comment by spydmobile** [comment]
+1
